.github/workflows/pull-request.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout base branch
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
You should note these changes in the PR description.

Reply:
They will be listed once the PR is ready. 
I am not able to convert it again to a draft PR.
Once it is ready I will communicate on slack and jira.